database/src/base/mod.rs

```diff
@@ -5,15 +5,49 @@ pub use crate::base::state_indexer::StateIndexerDbManager;
 pub mod tx_indexer;
 pub use crate::base::tx_indexer::TxIndexerDbManager;
 
+/// Additional database options for scylla database
+///
+/// `preferred_dc: Option<String>`,
+/// ScyllaDB preferred DataCenter
+/// Accepts the DC name of the ScyllaDB to filter the connection to that DC only (preferrably).
+/// If you connect to multi-DC cluter, you might experience big latencies while working with the DB.
+/// This is due to the fact that ScyllaDB driver tries to connect to any of the nodes in the cluster
+/// disregarding of the location of the DC. This option allows to filter the connection to the DC you need.
+/// Example: "DC1" where DC1 is located in the same region as the application.
+///
+/// `keepalive_interval: Option<u64>`
+/// ScyllaDB keepalive interval
+///
+/// `max_retry: u8`
+/// Max retry count for ScyllaDB if `strict_mode` is `false`
+///
+/// `strict_mode: bool`
+/// Attempts to store data in the database should be infinite to ensure no data is missing.
+/// Disable it to perform a limited write attempts (`max_retry`)
+/// before skipping giving up and moving to the next piece of data
+///
+/// `database_name: Option<String>`
+/// Postgres database name
+#[derive(Debug, Clone)]
+pub struct AdditionalDatabaseOptions {
+    #[cfg(feature = "scylla_db")]
+    pub preferred_dc: Option<String>,
+    #[cfg(feature = "scylla_db")]
+    pub keepalive_interval: Option<u64>,
+    #[cfg(feature = "scylla_db")]
+    pub max_retry: u8,
+    #[cfg(feature = "scylla_db")]
+    pub strict_mode: bool,
+    #[cfg(feature = "postgres_db")]
+    pub database_name: Option<String>,
```

Comment:
BTW why is `database_name` optional?

Reply:
we can use 2 types of configuration: 
1. specify all parameters individually 
```
DATABASE_URL="localhost"
DATABASE_USER="postgres"
DATABASE_PASSWORD="password"
DATABASE_NAME="near_data 
```
2. specify the connection string at once 
```
DATABASE_URL=postgres://postgres:password@localhost/near_data 
```

Reply:
cool, missed that 👍